backend/api/Areas/Tools/Helpers/ImportProjectsHelper.cs

```diff
@@ -108,45 +140,117 @@ private Entity.Project Merge(Entity.Project project, Model.ImportProjectModel mo
 
                 _logger.LogDebug($"Importing Project '{model.ProjectNumber}', agency:'{model.Agency}', workflow:'{model.Workflow}', status:'{model.Status}'");
 
+                var status = GetStatus(model.Status);
+
+                // Default to the last workflow this status is associated with.
+                // This isn't ideal and could end up with projects assocaited with an incorrect workflow, however
+                var workflow = !String.IsNullOrWhiteSpace(model.Workflow) ? GetWorkflow(model.Workflow) : model.Activity switch
```

Comment:
this is some funky syntax, so if model.workflow is null switch on the value of model.Activity?

Reply:
If they didn't provide the exact workflow in the JSON, then determine it based on the `Activity` value.